docs/deploying_server.md

```diff
@@ -71,13 +71,42 @@ zebra
 ```
 If everything is set up correctly, you will see 'zebra' prediction in the output.
 
-## Deploying Model Server on Baremetal
+## Deploying Model Server on Baremetal (without docker)
+It is possible to deploy Model Server outside of docker container.
+To deploy Model Server on baremetal, use pre-compiled binaries for your target OS from latest release.
+Alternatively it is possible to build package from scratch:
 
-There are two ways to start the server:
+```bash
+git clone https://github.com/openvinotoolkit/model_server
+cd model_server
+```
+
+The `ovms.tar.gz` package will appear in `dist/ubuntu` or `dist/redhat` directory.  
+Unpack the package:
```

Comment:
Here we should have a link placeholder how to download tar.gz.

But even if we have it here the instruction here is misleading -> it suggests that we will build the ovms from scratch but there is no instruction (or link) how to do it.

Reply:
I think first we should have instruction how to use binary distribution, and later on instruction/link how to build from source & extract the binary package from this process.

Reply:
Fixed